lib/hcc-config.cmake.in

```diff
@@ -22,6 +24,16 @@ set_target_properties(hsa-runtime64 PROPERTIES
   INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${HSA_HEADER}"
 )
 
+pkg_check_modules(LIBUNWIND REQUIRED libunwind)
+add_library(${LIBUNDWIND_LIBRARIES} INTERFACE IMPORTED)
```

Comment:
> However, those don't use the same cmake files. what am I missing?

The hcc-config.cmake uses the same targets, which is provided when doing `include( "${CMAKE_CURRENT_LIST_DIR}/hcc-targets.cmake" )`. This will included the targets that were installed/exported(as specified [here](https://github.com/RadeonOpenCompute/hcc/blob/clang_tot_upgrade/lib/CMakeLists.txt#L76)). It does not include the targets for external dependencies such as hsa. Rather, hsa is searched and created again in th hcc-config.cmake. The reason for this is to allow for the dependencies to be relocatable. We could link against the hsa library directly(in the build), however, this would use a hard-coded path in the target. So instead we create an imported target, and link against it. We define the imported target for the paths during build and define it again during hcc-config.cmake with the paths of the user's system.